docs/plugins/mapper-size.asciidoc

```diff
@@ -85,9 +81,4 @@ GET my-index-000001/_search
 <4> Uses a
```

Comment:
Do we want to remove this reference to `script_fields` too? It doesn't seem so useful, and I think we'll be phasing out `script_fields` now that we have runtime fields.

Reply:
I left it here to show an example of general access in scripts, the `script_field` seems to provide a simple way of giving that example. But I can remove it if you don't find it useful. I think I do.

Reply:
No strong preference from me. As a compromise we could just swap the order to emphasize `fields` is the preferred way to retrieve its value?

Reply:
Sounds like a good idea.